### PR TITLE
Completed css-two-column-layout challenge. -Paul Glorioso

### PR DESCRIPTION
**Pull request by PaulGlorioso**
Screenshot 2021-12-02 202910

### CODE REVIEW

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looks good, but there are some updates needed:
1. direct children of row classes should be column classes. you want to have two half columns in your rows, and those should contain the images and text elements
2. the images need to be adjust to match the example. I recommend wrapping the `img` tag and the `div` inside the `half-colunm` classes with a div so you can use flex to position everything.

**Comment by PaulGlorioso** [comment]
Screenshot 2021-12-03 100457

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        <div class="headp">
+          <p>
+            We've dedicated thousands of man hours to making healthcare
+            institutions in the Helium network run 10x more efficiently and
+            deliver an unparalleled quality of care.
+          </p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="imgcolumn">
```
Comment (BrettAlbright) [inline css-two-column-layout/index.html]: so the only thing I would add here is a `column-half` wrapper which would be the parent of the `imgcolumn` and `column`  classes. as this is a `two column` design, it is best for the main columns to be one half of the total row, and this also encapsulates the content nicely for each collection of elements in a column.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking great! Just a couple small formatting updates!

**Comment by PaulGlorioso** [comment]
Screenshot 2021-12-03 141158

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**